### PR TITLE
Added Player.addAllMounts and Player.addAllAddons

### PR DESCRIPTION
**Pull request by EPuncker**
Functions by @Xikini 

### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
- added Player.addAllMounts and Player.addAllAddons
- added doPlayerAddAddons to compat.lua as it was a very used function to addon npcs ([source](https://otland.net/search/1839600/?q=doPlayerAddAddons&o=relevance))

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

**Comment by EPuncker** [comment]
actually tested and working

**Review by omarcopires — APPROVED** [review]

**Comment by omarcopires** [comment]
There could also be added functionality for this, perhaps an admin talkaction to add to the player.

File: data/lib/core/player.lua [hunk 1]
```diff
@@ -358,3 +358,19 @@ end
 function Player.getTotalMoney(self)
 	return self:getMoney() + self:getBankBalance()
 end
+
+function Player.addAllAddons(self, addon)
```
Comment (DSpeichert) [inline data/lib/core/player.lua:362]: It sounds quite confusing when you're adding "all addons" but... need to provide addon as argument, no?

Maybe `addAddonToAllOutfits` is what it actually does?

Comment (EPuncker) [reply data/lib/core/player.lua:362]: yes, it add addon set 1 or 2, too all outfits

**Review by DSpeichert — APPROVED** [review]

**Pull request merged**